frontend/components/Companies/InvitationToJoin.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <div class="w-100% relative my-20 bg-zinc-800 ">
+    <div class="absolute inset-0">
+      <img class="h-full w-full object-cover" src="/images/lpm_22/lpm22-img1.webp" alt="">
+      <div class="absolute inset-0 bg-zinc-800 mix-blend-multiply" aria-hidden="true"/>
+    </div>
+    <div class="relative mx-auto max-w-7xl py-24 px-4 sm:py-32 sm:px-6 lg:px-8">
+      <div class="mx-auto max-w-7xl py-8 px-4 text-center sm:px-6 lg:py-10 lg:px-8">
+        <h2 class="tracking-tight text-white">
+          <span class="mb-7 block text-3xl">Wypromuj swoją firmę!</span>
+          <span class="block text-2xl">
+            Chciałbyś, aby Twoja firma brała udział w brandowych spotkaniach? <br>A może chcesz na stałe dołączyć do zespołu 
```

Comment:
```suggestion
            Chcesz, aby Twoja firma brała udział w brandowych spotkaniach? <br>A może masz chęć na stałe dołączyć do zespołu 
```

Reply:
Also I think that I'll remove `<br>` tag and use `<span>` or `<div>` to wrap this text.